ansible/templates/common/install.sh.j2

```diff
@@ -10,16 +10,17 @@ OMEROVER=${OMEROVER:-{{ omero_version }}}
 {% endif %}
 
 {% if doc %}
-curl -o OMERO.py.zip https://downloads.openmicroscopy.org/latest/omero5.3/py.zip
+cd {{ omero_user_home_dir }}
+wget https://downloads.openmicroscopy.org/latest/omero5.3/py.zip -O OMERO.py.zip
```

Comment:
If we use ``curl``, it is not a zip file that is actually returned.
i.e.  the output of ``file OMERO.py.zip`` is ``OMERO.py.zip: HTML document text``

wget returns the ``zip`` file.